packages/api-server/src/api/signup.ts

```diff
@@ -8,36 +8,36 @@ const router = Router()
 
 const alphaTest = new RegExp(/^[a-z0-9]+$/gi)
 
-const validateSubdomain = async (subdomain: string): Promise<boolean> => {
-  if (!alphaTest.test(subdomain)) {
+const validatePath = async (path: string): Promise<boolean> => {
+  if (!alphaTest.test(path)) {
     return false
   }
 
-  const id = await db.Server.getServerIdBySubdomain(subdomain.toLowerCase())
+  const id = await db.Server.getServerIdBySubdomain(path.toLowerCase())
 
   const signupRows = await db.dbQuery(
-    `SELECT id FROM server_signup WHERE subdomain = $1`,
-    [subdomain.toLowerCase()]
+    `SELECT id FROM server_signup WHERE path = $1`,
```

Comment:
You never updated this table. I would also keep it with the same `instance_path` that you used in the other table. You can add this migration to the same migration file I mentioned above.